### ISSUE TITLE
hoverctl start with --cache does not work

### ISSUE THREAD

**Issue by DonMartin76** [SpectoLabs/hoverfly#830]
## Description of the bug
When starting hoverfly using `hoverctl start` with a preloaded cached response file, it does not start correctly.

This is what happens:

```
$ hoverctl start --admin-port 8889 --proxy-port 8501 --cache ./test/mocked-responses.json 
Timed out waiting for Hoverfly to become healthy, returns status: 0
```

## Context information (for bug reports)

If I skip the `--cache` parameter and instead run this, it works:

```
hoverctl start --admin-port 8889 --proxy-port 8501
curl -X PUT -d '{"mode":"simulate"}' -H 'Content-Type: application/json' http://localhost:8889/api/v2/hoverfly/mode
curl -X PUT -d @./test/mocked-responses.json http://localhost:8889/api/v2/simulation
```

## Steps to reproduce the issue

1. Capture some data
2. Run `hoverctl start --cache /path/to/exported-data.json`
3. Error

### Expected result

I would expect hoverfly to start in simulate mode, using the JSON file provided via the `--cache` parameter as a base. Perhaps this is not correct?

## Additional relevant information
If not indicated above:
1. OS and version: e.g. macOS 10.14.4
2. Hoverfly version: 1.0.0.rc2

**Comment by tommysitu** [comment]
Hi @DonMartin76, a little bit dig into the code, I found this: https://github.com/SpectoLabs/hoverfly/blob/master/hoverctl/configuration/target.go#L86

The `--cache` flag is for supplying a boltdb cache (for caching request matching results) which is already deprecated. 

I believe you want to start the hoverfly with a simulation file. `hoverctl start` doesn't provide a flag to let you do that. It's quite odd. So you need to do `hoverctl import /path/to/exported-data.json` to import the simulation after hoverfly is started.

Alternatively, use `hoverfly` instead of `hoverctl` to start and import in one line: 
`hoverfly -import /path/to/exported-data.json -ap 8889 -pp 8501`

**Comment by DonMartin76** [comment]
Thanks for clarifying; I suspected something like that, and that I was having wrong assumptions on how that parameter works. I guess you can close this in that case? And perhaps remove the `--cache` parameter and/or add an `--import` parameter?

**Comment by tommysitu** [comment]
It may be possible to add `--import` flag. The `--cache` definitely needs to be thrown away. Let's keep this open for now.

**Comment by tommysitu** [comment]
The import flag for `hoverctl start` is available in [v1.0.1](https://github.com/SpectoLabs/hoverfly/releases/tag/v1.0.1)
